engine.py

```diff
@@ -103,7 +103,7 @@ def host_info(self):
                   engine.
         """
         version = self.shotgun.info().get("version") or ["unknown"]
-        return {"name": "Shotgun", "version": ".".join([str(x) for x in version])}
+        return {"name": "ShotGrid", "version": ".".join([str(x) for x in version])}
```

Comment:
I wasn't sure about changing this one. But I can't find anything that would break if we do so I made the change.

Reply:
Keep it to Shotgun, otherwise this will split the metrics for the engine.